test/copy_template_test.go

```diff
@@ -153,6 +154,7 @@ var _ = Describe("Copy template task", func() {
 
 			checkRemovedRecordsTemplate(newTemplate.Labels)
 			checkRemovedRecordsTemplate(newTemplate.Annotations)
+			Expect(newTemplate.OwnerReferences).To(HaveLen(1), "template should has owner reference")
```

Comment:
Can you check if the owner ref containes the expected Pod?

Reply:
done